processor/elastictraceprocessor/processor.go

```diff
@@ -52,6 +55,34 @@ func (p *Processor) Capabilities() consumer.Capabilities {
 }
 
 func (p *Processor) ConsumeTraces(ctx context.Context, td ptrace.Traces) error {
+	if isECS(ctx) {
+		resourceSpans := td.ResourceSpans()
+		for i := 0; i < resourceSpans.Len(); i++ {
```

Comment:
Here I think we'll need to go further and not only translate resource attributes, but also things on span. Or is that done in the exporter? E.g. I know db attributes are very different in the apm data model compared to otel - probably same for HTTP and many other things. It's ok to do it in follow ups - I think first covering resource attributes is enough.